build/yamls/base/antctl.yml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: antrea-agent
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antctl
+rules:
+  - nonResourceURLs:
+      - /apis/antctl.antrea.io/v1
```

Comment:
Could we use some generic naming instead of indicating this is for antctl only? Like clusterinfo, or controllerinfo/agentinfo or nodeinfo?
@tnqn: what you think?

Reply:
Agree, they should be named with what they are, instead of who use them.

Reply:
Thanks guys, but could you give me some suggestions for the name?
This endpoint was used for accessing antctl server side and we will add commands not related to info, like `support-bundle`.

Reply:
I think we could use clusterinfo or now (or controllerinfo and agentinfo if we want to separate controller and agent/node).

Not sure about support-bundle. What API groups K8s uses for logging/metrics, etc.?

Reply:
Changed to `clusterinfo.antrea.tanzu.vmware`.
The `metrics` in Kubelet does not have API group, it's a bare endpoint.